idseq_bench/scoring/idseq.py

```diff
@@ -71,7 +61,32 @@ def __init__(self, project_id, sample_id, pipeline_version, env='prod', local_pa
     self.pipeline_version = pipeline_version
     self.env = env
     self.store = local_path or STORE
+    self.set_directory_vars()
+
+  def set_directory_vars(self):
+    # this enables benchmark scoring to work with both SFN-WDL and original filepaths
+    env_dir = f"{{store}}idseq-samples-{{env}}"
```

Comment:
Now that these are no longer constants,  you can immediately apply `store`, `env`, `project_id`, `samples_id` and `pipeline_version` here, i.e. you do no need the `apply_context` anymore.

Reply:
Thanks! I modified to directly apply those into the variable and removed the apply_context() function.